allauth/socialaccount/providers/kakao/provider.py

```diff
@@ -8,7 +8,7 @@ class KakaoAccount(ProviderAccount):
     def properties(self):
         return self.account.extra_data.get('properties')
 
-    def get_avatar_url(self):
+    def get_profile_url(self):
```

Comment:
get_avatar_url is supposed to return the image, get_profile_url a link to the public viewable profile. So this change looks incorrect to me.